packages/omgidl-serialization/src/MessageReader.test.ts

```diff
@@ -492,8 +492,8 @@ module builtin_interfaces {
     writer.emHeader(true, 1, 8); // heightMeters emHeader
     writer.float64(data.heightMeters);
     writer.emHeader(true, 2, 4 + 4 + 1); // address emHeader
-    // dHeader for inner object not written again because the object size is already specified in the emHeader
-    writer.emHeader(true, 1, 1); // pointer emHeader
+    // dHeader for inner object not written again because the object size is already specified in the emHeader and the lengthCode of 5 allows it to not be written again
+    writer.emHeader(true, 1, 1, 5); // pointer emHeader
```

Comment:
is the reader supposed to be able to handle this both ways (length code ≥5 or <5)? If so, should we have another test which uses a length code <5?

(same applies to below test as well)

Reply:
Yeah that's correct. I'll add some more tests to account for this.